airflow/providers/docker/decorators/docker.py

```diff
@@ -144,7 +144,7 @@ def docker_task(
     return task_decorator_factory(
         python_callable=python_callable,
         multiple_outputs=multiple_outputs,
-        decorated_operator_class=_DockerDecoratedOperator,
+        decorated_operator_class=_DockerDecoratedOperator,  # type: ignore
```

Comment:
This one should not be needed any more @subkanthi. It has been fixed by @josh-fell  in #20235

Reply:
crediting the right person 😄 , it was suggested by @uranusjr

Reply:
Done, thanks